dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/Child.java

```diff
@@ -63,15 +64,21 @@ public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGit
 
         dockerfileGitHubUtil.createOrUpdateForkBranchToParentDefault(repo, fork, gitForkBranch);
 
+        RateLimiter rateLimiter = getRateLimiter();
```

Comment:
Since every command is going to need a rate limiter can we create the rate limiter outside of the commands and pass them to the command instance.